src/lfx/src/lfx/components/ollama/ollama.py

```diff
@@ -194,6 +210,9 @@ def build_model(self) -> LanguageModel:  # type: ignore[type-var]
             "verbose": self.verbose,
             "template": self.template,
         }
+        headers = self.headers
+        if headers is not None:
```

Comment:
Is it the case that headers will/ should never be provided with a local instance?